pkg/alertmanager/alertmanager.go

```diff
@@ -300,6 +302,16 @@ func clusterWait(position func() int, timeout time.Duration) func() time.Duratio
 	}
 }
 
+// withTenantIDFunc returns template.Option which adds to the templates additional function tenantID
+// returning the userID of the alertmanager tenant.
+func withTenantIDFunc(userID string) template.Option {
+	funcs := tmpltext.FuncMap{"tenantID": func() string { return userID }}
+	return func(text *tmpltext.Template, html *tmplhtml.Template) {
+		text.Funcs(funcs)
+		html.Funcs(tmplhtml.FuncMap(funcs))
```

Comment:
[nit] Should't be required here.

```suggestion
		html.Funcs(funcs)
```

Reply:
Thanks, fixed in https://github.com/grafana/mimir/pull/3758/commits/975295f6c74afad4ef5d2d47b8f68ba8fd34f48e